python/ray/experimental/serve/handle.py

```diff
@@ -28,11 +30,17 @@ def __init__(self, router_handle, endpoint_name):
         self.router_handle = router_handle
         self.endpoint_name = endpoint_name
 
-    def remote(self, *args):
-        # TODO(simon): Support kwargs once #5606 is merged.
+    def remote(self, *args, **kwargs):
+        if len(args) != 0:
```

Comment:
`#5606` is not merged yet, right? Does this still work?

Reply:
Yes. We have our own kwargs handling mechanism now. `Queue::enqueue_request` takes in `request_args: List, request_kwargs: Dict` now.